src/archive/zip.rs

```diff
@@ -27,11 +27,7 @@ use crate::{
 
 /// Unpacks the archive given by `archive` into the folder given by `output_folder`.
 /// Assumes that output_folder is empty
-pub fn unpack_archive<R, D>(
-    mut archive: ZipArchive<R>,
-    output_folder: &Path,
-    mut display_handle: D,
-) -> crate::Result<Vec<PathBuf>>
+pub fn unpack_archive<R, D>(mut archive: ZipArchive<R>, output_folder: &Path, mut out: D) -> crate::Result<Vec<PathBuf>>
```

Comment:
I find the name of the parameter `out` to be a little misleading just because one could assume it's the `out`_put_ of the function.

While it's actually our stdout control "thing", where we log stuff to the user.

Could you give this a less ambiguous name? Example: `log_out`.